### PR TITLE
Updates to changelogs for ADALv4 preview and MSALv2 preview releases

### PR DESCRIPTION
**Pull request by jennyf19**
I have not gone through the MSAL and ADAL issues to tag them with the release updates in the changelog. I don't know if everything listed has a corresponding issue...thoughts?

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,159 +0,0 @@
-Version 3.17.2
```
Comment (jennyf19) [inline changelog.txt:1]: this changelog.txt is outside adal and msal. I removed it as they each have their own corresponding changelog.txt file.

File: adal/changelog.txt [hunk 2]
```diff
@@ -1,3 +1,76 @@
+Version 4.0.0-preview
+==============
+This release includes:
+- Support MonoAndroid8.1 and Uap10.0 (Windows 10 applications)
+- Remove support for NetCore45 (.NET Core), which was for windows store applications (Win8), and Portable-net45+win (PCL)
```
Comment (jmprieur) [inline adal/changelog.txt]: NetCore45 is not .NET Core.

File: adal/changelog.txt [hunk 3]
```diff
@@ -1,3 +1,76 @@
+Version 4.0.0-preview
+==============
+This release includes:
```
Comment (henrik-me) [inline adal/changelog.txt:3]: >his release includes: [](start = 1, length = 21)

IMO: both this and MSAL notes should reference the blog post via a link to the wiki? #Closed

**Review by henrik-me — APPROVED** [review]
:shipit:

File: adal/changelog.txt (hunk 3 above)
Comment (jennyf19) [reply adal/changelog.txt:3]: Agree. I will make the change accordingly.

---
In reply to: [213016360](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1230#discussion_r213016360) [](ancestors = 213016360)

File: adal/changelog.txt [hunk 4]
```diff
@@ -1,3 +1,76 @@
...
+- Compatibility with the ADAL.NET V3 cache, but also have serialization with the new Unified cache format, bringing compatibility with MSAL.NET V2.x and being common on all platforms. 
+   Note that on mobile platforms (UWP, Xamarin.iOS, Xamarin.Android), you will benefit automatically from the token cache sharing between ADAL and MSAL as the library handles the serialization
+   to the container of choice of the platform (respectively isolated storage, iOS key chain, and Android shared preferences).
+- In the Xamarin.iOS platform, AuthenticationContext has a new property named KeychainSecurityGroup. This Xamarin iOS specific property enables you to direct the application to share the 
+   token cache with other applications sharing the same keychain security group. If you provide this key, you must add the capability to your Application Entitlement. 
+   For more info, see https://aka.ms/adal-net-sharing-cache-on-ios. This API may change in a future release.
+-To preserve the single-sign-on (SSO) state, the new versions of ADAL(v4) and MSAL(v2) share the same token cache, are capable of reading the ADAL 3.x token cache
```
Comment (jmprieur) [inline adal/changelog.txt]: Isn't this similar to line 7?
Do you want to take inspiration from the blog post?

**Review by jmprieur — COMMENTED** [review]
- Not sure if there is not a dupe between line 7 and 13
- netcore45 is really win8 (not .NET Core. I know this is confusing)
otherwise LGTM
you might want to provide the aka.ms link to the blog post?

File: adal/changelog.txt (hunk 2 above)
Comment (jennyf19) [reply adal/changelog.txt]: I copied this from the blog post table 2.1...I will remove the (.NET Core) part...should the blog post be updated as well?

---
In reply to: [213016249](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1230#discussion_r213016249) [](ancestors = 213016249)

File: adal/changelog.txt (hunk 4 above)
Comment (jennyf19) [reply adal/changelog.txt]: I think this is from the blog post, but let me move things around a bit.

---
In reply to: [213026350](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1230#discussion_r213026350) [](ancestors = 213026350)

Comment (henrik-me) [reply adal/changelog.txt]: My thought was to reference the blog post as it will be a wiki entry.

Comment (jmprieur) [reply adal/changelog.txt]: only a reference to the blog post? @henrik-me ? that can do.

File: adal/changelog.txt (hunk 2 above)
Comment (henrik-me) [reply adal/changelog.txt]: IMO, should remove the NetCore45 parts of the sentence and remove the () around win8. Perhaps we should simply call it Windows 8 and Windows Phone 8?

---
In reply to: [213028783](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1230#discussion_r213028783) [](ancestors = 213028783,213016249)

Comment (jennyf19) [reply adal/changelog.txt]: updated.

---
In reply to: [213053801](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1230#discussion_r213053801) [](ancestors = 213053801,213028783,213016249)

**Pull request merged**